### PR TITLE
Update mucommander to 0.9.3

### PR DESCRIPTION
**Pull request by yurikoles**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by yurikoles** [comment]
It seems like `mucommander` only works with `java8`.

**Comment by ran-dall** [comment]
@yurikoles Take a look at https://github.com/Homebrew/homebrew-cask/issues/16383, I believe it's something that's coming but not released yet. From my little understanding, it's something that core Homebrew needs compatibility with first before we can do something about it.

**Comment by yurikoles** [comment]
@ran-dall thanks for suggestion, I see that is actually possible now, fixed.
```depends_on java: '1.8'```

File: Casks/mucommander.rb [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+  version '0.9.3'
+  sha256 'cb74e0f857ff5bac804be0548e331de0dbfb9fa74acf948a095bdf2b256a4b34'
 
   # github.com/mucommander/mucommander was verified as official when first introduced to the cask
   url "https://github.com/mucommander/mucommander/releases/download/#{version}/mucommander-#{version}.dmg"
   appcast 'https://github.com/mucommander/mucommander/releases.atom'
   name 'muCommander'
   homepage 'https://www.mucommander.com/'
 
+  depends_on java: '1.8'
```
Comment (vitorgalvao) [inline Casks/mucommander.rb]: This does not work.

**Pull request merged**